### DISCUSSION TITLE
💡자체 Build Test를 구현했습니다!

### DISCUSSION THREAD

**Discussion by jongwooo** [grooteogi/frontend#82] [Announcements]
## 개요
Vercel에서만 확인 가능했던 Build Test를 Github Action으로 구현했습니다.

## 작업내용
- Push 이벤트 발생 시 포크된 레포지토리에서 Build 진행
<img width="843" alt="스크린샷 2022-05-23 오후 3 10 48" src="https://user-images.githubusercontent.com/44025432/169755010-30f43876-a156-4296-b44e-ddf4b215e5f2.png">

- Build 실패 시 Comment 알림 생성
<img width="863" alt="스크린샷 2022-05-23 오후 3 11 01" src="https://user-images.githubusercontent.com/44025432/169755044-5069e115-440a-46ac-a140-dd3d31c36d4c.png">

## 주의사항
@Chloekkk @Kimsj912 @hong-soohyuk 포크된 레포지토리에서 실행되므로 각 레포지토리마다 환경변수 설정이 필요합니다.

**Comment by Kimsj912** [comment]
와아아 좋아요 !!!! 수고많으셨습니다ㅏ
